src/lib/parser_txdef.c

```diff
@@ -90,9 +90,11 @@ void parser_txInit(parser_tx_t *tx) {
     tx->feesLen = 0;
     parser_feesInit(&tx->fees);
 
+    tx->multisigPtr = NULL;
+    tx->multisigLen = 0;
+    parser_multisigInit(&tx->multisig);
+
     tx->sendmsgPtr = NULL;
     tx->sendmsgLen = 0;
     parser_sendmsgInit(&tx->sendmsg);
-
-    parser_multisigInit(&tx->multisig);
```

Comment:
Just a question: why did you change the order? What is `tx->sendmsg`?

Reply:
Just to keep the same order as in the struct definition.
It is not a functional change